src/types.ts

```diff
@@ -216,16 +230,26 @@ export type TransactionBody = {
     metadataHash?: FixlenBuffer<typeof METADATA_HASH_LENGTH>,
     validityIntervalStart?: Uint,
     mint?: Mint,
+    scriptDataHash?: FixlenBuffer<typeof SCRIPT_DATA_HASH_LENGTH>,
+    collaterals?: Collateral[],
+    requiredSigners?: RequiredSigner[],
+    networkId?: Uint,
 }
 
 export type Transaction = {
     body: TransactionBody,
     witnessSet: Unparsed,
+    scriptValidity?: Unparsed,
     auxiliaryData: Unparsed,
 }
 
+// raw tx items
+// https://github.com/input-output-hk/cardano-node/blob/54119c80057f88af5acdd7d54969dd461e4cf26e/cardano-api/src/Cardano/Api/TxBody.hs#L1591-L1598
 export type RawTransaction = {
```

Comment:
Is this deprecated?

Reply:
It is. I've added deprecation comments to some relevant places (mainly user-facing docs) in https://github.com/vacuumlabs/cardano-hw-interop-lib/pull/3/commits/ff37c6a1d0a4833840409409e638dde6a06b0111.